.github/workflows/pr-actions.yml

```diff
@@ -118,3 +118,13 @@ jobs:
           gh pr comment $PR_NUM -b "fix:${PR_ACTION} failed or was cancelled. For details, see $GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID."
         env:
           GH_TOKEN: ${{ secrets.OPENTELEMETRYBOT_GITHUB_TOKEN }}
+
+  rerun_pr_tests:
+    name: rerun_pr_tests
+    if: ${{ github.event.issue.pull_request }}
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: estroz/rerun-actions@main
```

Comment:
can you pin this to a version or even better to a hash? Pulling from main from a repo that has not been updated 4 years sounds risky

Reply:
Thought about that, though the logic seems simple enough. Another option would be to rebuild the action ourselves, I guess. In any case, It's pinned now.